src/OpenLoco/CMakeLists.txt

```diff
@@ -558,10 +558,6 @@ if (NOT WIN32)
     add_dependencies(OpenLoco segfiles)
 endif()
 
-if (NOT APPLE AND NOT MSVC)
-    target_link_libraries(OpenLoco stdc++fs)
```

Comment:
I'm surprised we don't need this anymore. I thought we were using too old a gcc but clearly we do have gcc >= 9 now.

Reply:
I think on the Flatpak builder it is using version 12.